css/particle-modeler.less

```diff
@@ -46,6 +53,18 @@
       background-color: white;
     }
   }
+  .restart-button{
+    position: fixed;
+    top: 10px;
```

Comment:
The restart button doesn't move with the modeler window, so if you make your browser window taller and the modeler is down in the middle, the button is way up at the top and partially cut off by a black border.